Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/NotificationUtils.java

```diff
@@ -387,7 +387,6 @@ public static void showNotification(Context context, String title, String msg, J
                 
         if (TRIP_STARTED.equals(notificationType)) {
             if (key.equals("USER")) {
-                startMediaPlayer(context,R.raw.notify_otp_sound, true);
```

Comment:
why is this removed?

Reply:
This is not required. normal notification sound itself is working fine.

Reply:
what kind of sound notify_otp play?